package.json

```diff
@@ -112,14 +114,14 @@
 	"scripts": {
 		"build": "tsc && npm run download:files && npm run commit:hash && cp -r public dist",
 		"commit:hash": "git rev-parse HEAD > data/LAST_API_COMMIT_HASH.txt",
-		"coverage": "npm run clean && NODE_ENV=test FAKE_PROBE_IP=api NEW_RELIC_ENABLED=false NEW_RELIC_LOG_ENABLED=false c8 mocha",
+		"coverage": "npm run clean && NODE_ENV=test TEST_MODE=unit NEW_RELIC_ENABLED=false c8 mocha",
 		"clean": "rimraf coverage",
 		"download:files": "tsx src/lib/download-files.ts",
 		"prepare": "npm run download:files; husky install || echo 'Failed to install husky'",
-		"stats": "NEW_RELIC_ENABLED=false NEW_RELIC_LOG_ENABLED=false tsx probes-stats/known.ts",
-		"start": "NODE_ENV=production node --max_old_space_size=3584 --max-semi-space-size=128 --experimental-loader newrelic/esm-loader.mjs -r newrelic dist/src/index.js",
```

Comment:
Yes, `NODE_ENV=test` should be passed to the e2e API instance. Also `ENV NODE_ENV production` already exists in Dockerfile so that looks like duplication.